documentation/waap/waap-rules/advanced-rules.md

```diff
@@ -41,8 +41,8 @@ The advanced rule object contains the following properties:
     "source": "string",
     "enabled": true,
     "description": "string",
-    "phase": "EXECUTION_PHASE_UNSPECIFIED",
-    "ruleAction": {
+    "phase": "access",
+    "action": {
         "allow": {},
```

Comment:
Only one action can be used. If a user provides several actions like "allow" + "block", the API will return an error saying that the only one action is allowed